packages/pipeline/test/parsers/paradex_orders/index_test.ts

```diff
@@ -52,3 +52,4 @@ describe('paradex_orders', () => {
         });
     });
 });
+// tslint:enable:custom-no-magic-numbers
```

Comment:
`tslint:enable` and `tslint:disable` operate on a per-file basis. So you don't need this comment. https://palantir.github.io/tslint/usage/rule-flags/